aten/src/ATen/native/cuda/BatchLinearAlgebra.cu

```diff
@@ -916,14 +920,14 @@ AT_ERROR("lu: MAGMA library not found in "
 
   if (self.dim() == 2) {
     // If `pivots` is defined, then we have to compute them.
-    // magmaLu and magmaLuNoPiv use a hybrid CPU-GPU algorithm to compute
+    // cu_xgetrf and magmaLuNoPiv use a hybrid CPU-GPU algorithm to compute
```

Comment:
Is this how cusolverDnDgetrf works?

Reply:
I just did a text-based replacement. Will reword these comments.